src/codegen/codegen_cpp_visitor.hpp

```diff
@@ -842,6 +842,15 @@ class CodegenCppVisitor: public visitor::ConstAstVisitor {
     /*                             Code-specific helper routines                            */
     /****************************************************************************************/
 
+    /**
+     * Add the variable tqitem during `get_int_variables`.
+     */
+    virtual void add_variable_tqitem(std::vector<IndexVariableInfo>& variables) = 0;
+
+    /**
+     * Add the variable point_process during `get_int_variables`.
+     */
+    virtual void add_variable_point_process(std::vector<IndexVariableInfo>& variables) = 0;
```

Comment:
Question: do we have any other code generators that inherit directly from `CodegenCppVisitor` (other than the NEURON and coreNEURON ones)? If we don't, LGTM.

Reply:
this is the "correct" place to lift it to.